.github/workflows/tests.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install -g yarn
+    - run: yarn && yarn add --frozen-lockfile node-rdkafka@2.7.4
+    - name: Start Docker containers for Zookeeper, Kafka and Schema Registry
+      run: cd kafka-setup && docker-compose up -d
```

Comment:
can we use the kafka:start command? https://github.com/nodefluent/kafka-streams/blob/master/package.json#L28

Reply:
Yep. This command is for local dev and deals with cleaning up resources etc.